src/basic_recipes/poly.jl

```diff
@@ -122,6 +133,9 @@ function plot!(plot::Poly{<: Tuple{<: Union{Polygon, AbstractVector{<: PolyEleme
         color = plot.color,
         colormap = plot.colormap,
         colorrange = plot.colorrange,
+        lowclip = plot.lowclip,
```

Comment:
wireframe doesn't use those ;)

Reply:
But that function is calling mesh. Or is it doing something in the background to also call wireframe?

Reply:
Oh sorry, I guess you're right... the diff view made it look that way:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/1010467/180826997-2f8a64dc-f1d2-474f-bb79-87fdb4d4171b.png">

Reply:
Oh, that is missleading :P haha